src/stories/icon/icon-heart/IconHeart.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import { ComponentStory, ComponentMeta } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import { IconHeart } from "./IconHeart";
+
+type IconHeartProps = typeof IconHeart;
+
+export default {
+  title: "Atoms / Icons",
+  component: IconHeart,
+  decorators: [withDesign],
+  parameters: {
+    design: {
+      type: "figma",
+      url:
+        "https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?node-id=1385%3A10755",
```

Comment:
I do not think this is the most precise link for the component. If I go to Figma, locate the wishlist icon in the left hand side, click "Copy link" I get https://www.figma.com/file/Zx9GrkFA3l4ISvyZD2q0Qi/Designsystem?node-id=656%3A5407